src/Adapter/ApiAccess/CommandHandler/AddApiAccessHandler.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+/**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ */
+
+declare(strict_types=1);
+
+namespace PrestaShop\PrestaShop\Adapter\ApiAccess\CommandHandler;
+
+use PrestaShop\PrestaShop\Core\CommandBus\Attributes\AsCommandHandler;
+use PrestaShop\PrestaShop\Core\Domain\ApiAccess\Command\AddApiAccessCommand;
+use PrestaShop\PrestaShop\Core\Domain\ApiAccess\CommandHandler\AddApiAccessCommandHandlerInterface;
+use PrestaShop\PrestaShop\Core\Domain\ApiAccess\ValueObject\ApiClientId;
+use PrestaShopBundle\Entity\ApiAccess;
+use PrestaShopBundle\Entity\Repository\ApiAccessRepository;
+
+#[AsCommandHandler]
+class AddApiAccessHandler implements AddApiAccessCommandHandlerInterface
+{
+    private ApiAccessRepository $repository;
+
+    public function __construct(ApiAccessRepository $repository)
+    {
+        $this->repository = $repository;
+    }
+
+    public function handle(AddApiAccessCommand $command): ApiClientId
+    {
+        $apiAccess = new ApiAccess();
+        $apiAccess->setClientId($command->getApiClientId()->getValue());
+        $apiAccess->setClientName($command->getClientName());
+        $apiAccess->setActive($command->isEnabled());
```

Comment:
```suggestion
        $apiAccess->setEnabled($command->isEnabled());
```
We should probably rename this field, but it is not mandatory It can be done later as well

Reply:
Put 'enabled' everywhere ( entity, behat tests, cqrs ... )